config/prod/bridgeserver2.yaml

```diff
@@ -34,8 +26,6 @@ parameters:
   HibernateConnectionUsessl: 'true'
   NewRelicAppName: bridgeserver2-prod
   NewRelicLicenseKey: !ssm /infra/NewRelicLicenseKey
```

Comment:
aren't you going to setup roles to access newrelic and synapse keys as well?

Reply:
No, because those changes are neither urgent nor are they related to the AWS security issue.